tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import re
+
+import pytest
+import random
+
+from tests.helpers import get_worker_nodes
+
+from ocs_ci.framework.pytest_customization.marks import tier4b
+from ocs_ci.ocs.resources import pod
+
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs import node
```

Comment:
please import all in 1 line:
from ocs_ci_ocs import (
    machine, constants, ocp, node
)

Reply:
Done